### PR TITLE
Description

### PR DESCRIPTION
**Pull request by mltbnz**
## Additions

* description widget
* description form (because of pickers and some other reasons I choose to use a dedicated view)

### CODE REVIEW

File: Wegli/App/Redux/AppState.swift [hunk 1]
```diff
@@ -15,13 +15,36 @@ import UIKit
...
+    // MARK: Description
+    struct Car {
+        var color: String?
+        var type: String?
+        var licensePlateNumber: String?
+    }
+    var car = Car()
+    
+    struct Crime {
+        static let crimes = ["Stand auf der Radspur", "Core Data", "Core Data", "Core Data"]
```
Comment (mltbnz) [inline Wegli/App/Redux/AppState.swift]: @phoet I could use some input on the input values. Is there something I could use already in the old app?

Comment (phoet) [reply Wegli/App/Redux/AppState.swift]: https://github.com/weg-li/weg-li/blob/master/app/lib/vehicle.rb#L165-L212

Comment (phoet) [reply Wegli/App/Redux/AppState.swift]: It's not a crime btw

Comment (mltbnz) [reply Wegli/App/Redux/AppState.swift]: haha. I am open for better naming. But thanks for the hint

Comment (mltbnz) [reply Wegli/App/Redux/AppState.swift]: I'll use `charges` 👍

Comment (mltbnz) [reply Wegli/App/Redux/AppState.swift]: @phoet is there also a reference on how the email looks like?

Comment (phoet) [reply Wegli/App/Redux/AppState.swift]: Of course, but the easiest way is to use weg-li and look at the preview.

**Pull request merged**